application/models/TemplateConfig.php

```diff
@@ -1007,10 +1075,11 @@ public function getAllDbTemplateFolders()
     }
 
     /**
-     * Returns an array with uninstalled or incompatible survey themes
-     * @return array|null
+     * Returns an array with uninstalled and/or incompatible survey themes
+     * @return array
```

Comment:
If you wanna make future Psalm analysis happy, you can be more precise in your annotations than `array`, like `TemplateConfig[]` or `array<string, TemplateConfig>`, whatever is the correct type (didn't read the function body).